agents/node/vcxagent-core/vcx-agent.js

```diff
@@ -106,11 +143,11 @@ async function createVcxAgent ({ agentName, protocolType, agencyUrl, seed, webho
   async function createInvite (connectionName) {
     const connection = await createConnection(connectionName)
     const invite = await connection.inviteDetails()
-    return { invite, connection }
+    return {invite, connection}
```

Comment:
Are we changing linter rules?

Reply:
I've run linter on whole agent codebase, there was bunch of other issues apart from this.